tests/manage/mcg/test_write_to_bucket.py

```diff
@@ -69,9 +69,69 @@ def test_write_file_to_bucket(self, mcg_obj, awscli_pod, bucket_factory):
     @pytest.mark.polarion_id("OCS-1949")
     @tier1
     @acceptance
-    def test_data_reduction(self, mcg_obj, awscli_pod, bucket_factory):
+    def test_data_deduplication(self, mcg_obj, awscli_pod, bucket_factory):
         """
         Test data reduction mechanics
+        Args:
+            mcg_obj (obj): An object representing the current state of the MCG in the cluster
+            awscli_pod (pod): A pod running the AWSCLI tools
+            bucket_factory: Calling this fixture creates a new bucket(s)
+
+        """
+        download_dir = '/aws/deduplication/'
+        for i in range(2, 4):
```

Comment:
why `2, 4`?

Reply:
the code began a bit differnetly where i would download specific files from our files bucket, i needed specific files with the numbers 2,3 in them but the code changed, i'll change this as well.